tests/vcs/git/GitCommitBearTest.py

```diff
@@ -100,6 +100,34 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_github_merge_commit(self):
+        run_shell_command('echo "a" >> testfile1.txt ')
+        run_shell_command('git add testfile1.txt')
+        self.git_commit('Add testfile1')
+        commit_hash1, _ = run_shell_command('git rev-parse HEAD')
+        commit_hash1 = commit_hash1.strip('\n')
+
+        run_shell_command('echo "b" >> testfile2.txt ')
+        run_shell_command('git add testfile2.txt')
+        self.git_commit('Add testfile2"')
+        commit_hash2, _ = run_shell_command('git rev-parse HEAD')
+        commit_hash2 = commit_hash2.strip('\n')
+
+        magic_github_merge_commit = ('Merge ' +
+                                     str(commit_hash2) + ' into ' +
+                                     str(commit_hash1) + '\n')
+        run_shell_command('echo "c" >> testfile3.txt ')
+        run_shell_command('git add testfile3.txt')
+        self.git_commit(magic_github_merge_commit)
+
+        self.assertEqual(self.run_uut(), [])
```

Comment:
To prove that this is working, you need to get an error out of the bear.  An error which proves it is looking at the commits before the 'Travis merge commit'.